processor/tailsamplingprocessor/internal/sampling/ottl_test.go

```diff
@@ -0,0 +1,122 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sampling
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/component/componenttest"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl"
+)
+
+func TestEvaluate_OTTL(t *testing.T) {
+	traceID := pcommon.TraceID([16]byte{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16})
+
+	cases := []struct {
+		Desc                string
+		SpanConditions      []string
+		SpanEventConditions []string
+		Spans               []spanWithAttributes
+		Decision            Decision
+	}{
+		{
+			// policy
+			"OTTL conditions not set",
```

Comment:
Should this be an error condition? At least one of the conditions should be specified, I suppose.

Reply:
I referred to the impl of filterprocessor: 
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/be4b4efbb2dd32c9b1c7ded8720979fdeac33cd2/processor/filterprocessor/traces.go#LL45C3-L45C3

I think it's still good to stop user from using a blank condition(mistakenly). But I am not confident about the necessity to change it since we have validation at the begining of  `Evaluate`:
```go
	if ocf.sampleSpanExpr == nil && ocf.sampleSpanEventExpr == nil {
		return NotSampled, nil
	}
```

Feel free to let me know and I am ready to change the init function to return an `error` when empty ottl condition is set.